api/specs/web-server/_catalog.py

```diff
@@ -34,6 +36,33 @@
 #
 
 
+@router.get(
+    "/dev/catalog/services/-/latest",
```

Comment:
I don't quite understand the use of `dev` here. Does it mean it is currently in development and will be moved to `/catalog/services/-/latest` once it is ready for production?

Reply:
yes. This is a trick I use to have both entries (the deprecated and the new one) for a while until the front-end is adapted and the backend fully connected.